test/fuzzing/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env sh
+
+set -eu
+
+export FUZZER_DIR=$SRC/linkerd2/test/fuzzing
+
+compile_go_fuzzer github.com/linkerd/linkerd2/pkg/profiles FuzzProfilesValidate FuzzProfilesValidate
+compile_go_fuzzer github.com/linkerd/linkerd2/pkg/profiles FuzzRenderProto FuzzRenderProto
+compile_go_fuzzer github.com/linkerd/linkerd2/controller/api/destination FuzzAdd FuzzAdd
+compile_go_fuzzer github.com/linkerd/linkerd2/controller/api/destination FuzzGet FuzzGet
+compile_go_fuzzer github.com/linkerd/linkerd2/controller/api/destination FuzzGetProfile FuzzGetProfile
+compile_go_fuzzer github.com/linkerd/linkerd2/controller/api/destination FuzzProfileTranslatorUpdate FuzzProfileTranslatorUpdate
+compile_go_fuzzer github.com/linkerd/linkerd2/pkg/healthcheck FuzzFetchCurrentConfiguration FuzzFetchCurrentConfiguration
+compile_go_fuzzer github.com/linkerd/linkerd2/pkg/inject FuzzInject FuzzInject
+compile_go_fuzzer github.com/linkerd/linkerd2/pkg/identity FuzzServiceCertify FuzzServiceCertify
+compile_go_fuzzer github.com/linkerd/linkerd2/test/fuzzing FuzzParseContainerOpaquePorts FuzzParseContainerOpaquePorts
+compile_go_fuzzer github.com/linkerd/linkerd2/test/fuzzing FuzzParsePorts FuzzParsePorts
+compile_go_fuzzer github.com/linkerd/linkerd2/test/fuzzing FuzzHealthCheck FuzzHealthCheck
```

Comment:
Is there any point for this file to exist in this repo if it depends on functions defined in the other repo? I.e. can we omit this file and keep it in the other repo?

Reply:
I would suggest putting this content here: https://github.com/google/oss-fuzz/blob/master/projects/linkerd2/build.sh this also makes sense as `compile_go_fuzzer` is a script on the oss-fuzz infrastructure

Reply:
CC @AdamKorcz

Reply:
I'll make this change as part of google/oss-fuzz#7609.